Model source installatoion readme instead of file name (#1375)
AUTOMATIC1111/stable-diffusion-webui: ESRGAN/readme.md

>  * You have to install `ESRGAN` models in this folder for upscaling.

Reviewer: ```suggestion
 * Place `ESRGAN` models in this folder for upscaling.
```

Reviewer: Actually, it looks like this file should probably move to `models/ESRGAN` now.

Author: Fixed in https://github.com/AUTOMATIC1111/stable-diffusion-webui/pull/1375/commits/e35521f36684498cccd6aec4bcee50b8156fc989
